ha/test/main.py

```diff
@@ -25,9 +25,12 @@
 import os
 import pathlib
 import sys
-import pytest
+import unittest
```

Comment:
pytest package is replaced with unittest right ? are any issues seen in installation of this unititest ?

Reply:
'unittest' comes packaged with Python, no need to install it separately.